localstack/utils/bootstrap.py

```diff
@@ -51,6 +55,22 @@
     "firehose": ["kinesis"],
     "transcribe": ["s3"],
 }
+
+# Optional dependencies of services on other services
+# - maps from API names to list of other API names that they _optionally_ depend on: <service>:<dependent-services>
+# - an optional service dependency is a service without which a service's basic functionality breaks,
```

Comment:
```suggestion
# - an optional service dependency is a service without which a service's basic functionality doesn't break,
```

Reply:
but I thought the point is that the basic functionality of a service does not break if the service it optionally  depends on is disabled?

Reply:
You are totally right 😄